helix-core/Cargo.toml

```diff
@@ -41,3 +41,5 @@ chrono = { version = "0.4", default-features = false, features = ["alloc", "std"
 
 [dev-dependencies]
 quickcheck = { version = "1", default-features = false }
+fern = "0.6"
+chrono = { version = "0.4", default-features = false, features = ["clock"] }
```

Comment:
Are these two imports still necessary?

Reply:
Yeah, they were necessary for seeing logs with unit tests. Though it might be worthwhile to leave this out until we set up common unit testing code for things like this.

Reply:
Let's remove this for now, I personally either use a debugger or switch to `println!()` when debugging then remove it before commiting